### PR TITLE
chore: Migrate DpDetails stories to v7.

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://github.com/demos-europe/demosplan-ui/pull/379

- Migrate DpDetails stories to v7.

### CODE REVIEW

File: src/components/DpDetails/DpDetails.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    title: "Components/Details",
+    argTypes: {},
+    render: (args) => ({
+        components: {
+            DpDetails,
+        },
+        setup() {
+            return { args }
+        },
+        template: `<dp-details primary secondary v-bind="args">
```
Comment (hwiem) [inline src/components/DpDetails/DpDetails.stories.tsx:15]: I think `primary` and `secondary` should be removed here

File: src/components/DpDetails/DpDetails.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpDetails from './DpDetails.vue'
+
+const meta: Meta<typeof DpDetails> = {
+    component: DpDetails,
+    title: "Components/Details",
+    argTypes: {},
```
Comment (hwiem) [inline src/components/DpDetails/DpDetails.stories.tsx:7]: Do we need to pass `argTypes` if it's empty?

File: src/components/DpDetails/DpDetails.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            return { args }
+        },
+        template: `<dp-details primary secondary v-bind="args">
+        Content to be revealed onClick of the summary.
+        </dp-details>`,
+    })
+}
+
+export default meta
+type Story = StoryObj<typeof DpDetails>
```
Comment (hwiem) [inline src/components/DpDetails/DpDetails.stories.tsx:22]:
```suggestion
export default meta

type Story = StoryObj<typeof DpDetails>
```

File: src/components/DpDetails/DpDetails.stories.tsx (hunk 2 above)
Comment (ahmad-demos) [reply src/components/DpDetails/DpDetails.stories.tsx]: Done in https://github.com/demos-europe/demosplan-ui/pull/400/commits/9ececfdfdb0bb23ebedd09097c8aac43c61d79d6

File: src/components/DpDetails/DpDetails.stories.tsx (hunk 1 above)
Comment (ahmad-demos) [reply src/components/DpDetails/DpDetails.stories.tsx]: Done in https://github.com/demos-europe/demosplan-ui/pull/400/commits/9ececfdfdb0bb23ebedd09097c8aac43c61d79d6

**Review by hwiem — APPROVED** [review]
lgtm, code review only

**Pull request merged**